functions/backup.sh

```diff
@@ -65,7 +62,7 @@ create_backup_config() {
   
   mkdir -p ${confdir}
   touch ${confdir}/tapelist
-  hostname=`/bin/hostname`
+  local hostname=`/bin/hostname`
```

Comment:
The newer form `$(cmd)` is used elsewhere in this file, rather than the deprecated form `` `cmd` ``.